src/main/java/wolox/training/models/Book.java

```diff
@@ -37,6 +42,7 @@
     private String title;
 
     @NotNull
+    @Column(name = "subtitle")
```

Comment:
Why do you need this change?

Reply:
Because the database was generating other fields with this attribute (sub_title, sub_ti). If we have a specific name, we avoid the problem.